.rubocop.yml

```diff
@@ -11,19 +11,13 @@ AllCops:
 
 # ----- Layout ------
 
-Layout/LineLength:
-  Max: 157
-  Exclude:
-    - lib/cocina/models/*
-    - 'lib/cocina/normalizers/mods/title_normalizer.rb'
-    - 'spec/cocina/mapping/descriptive/h2/related_item_h2_spec.rb'
-    - 'spec/cocina/mapping/descriptive/mods/physical_description_spec.rb'
-    - 'spec/cocina/mapping/descriptive/mods/title_info_spec.rb'
-
 Layout/BeginEndAlignment: # (new in 0.91)
   Enabled: true
 Layout/EmptyLinesAroundAttributeAccessor:
   Enabled: true
+Layout/LineLength:
+  Exclude:
+    - lib/cocina/models/*
```

Comment:
- 157 is an odd linelength to choose as max
- this was going to have so many exceptions due to longer class names that it became pointless.
- without excepting `lib/cocina/models/*` which was previous excepted, the max length was something like 1875